docs/_data/groups/2017.yml

```diff
@@ -143,6 +143,14 @@
     - name: Gabriel Somoza
       email: gabriel@strategery.io
 
+- name: PHP Varna
+  location: Varna, Bulgaria
+  meetup: https://www.meetup.com/phpvarna/
+  twitter: https://twitter.com/phpvarna
+  contacts:
+    - name: Boyan Yordanov
+      email: message trough meetup or twitter
```

Comment:
Please remove the `email` property. If we leave it, it will create a `mailto:` link, and I would rather visitors simply click on your Meetup or Twitter links instead.

Reply:
@ramsey We actually have a number of groups listed with "contact through website/meetup" already in place, and the mailto links do exactly what you describe. We ought to standardize on how we want to indicate this. I agree that "mailto: contact through Meetup" doesn't really work, but not sure of the best way to render this in the current template. Thoughts?

Reply:
I don't mind leaving the email property with these messages. The messages will only show up when you click them and they open your email program (or whatever is configured to open for `mailto` links).

If we drop them, though, the email icons/links go away, and I think people will assume to click on the Meetup/Twitter links to try to make contact through those.

Reply:
@ramsey @boyanyordanov Makes sense to remove them, then. Unless there's any objection, I'll accept Boyan's PR as-is, then go back and remove all the cases of "contact through Meetup/etc.", so we don't have to involved several other people.

Reply:
I am sorry that I didn't reply earlier. I copy-pasted that from another entry. I can remove it. And perhaps remove all other instances while I am at it?

Reply:
@boyanyordanov No problem! I had started editing the other instances, but was interrupted, so I'll finish those, but if you want to remove your instance, go right ahead and I'll get everything else synced up a bit later today. Thanks!